doc/source/building/working_with_images/vagrant_setup.rst

```diff
@@ -0,0 +1,134 @@
+.. _setup_vagrant:
+
+KIWI Image Description for Vagrant
+==================================
+
+.. sidebar:: Abstract
+
+   This page provides further information for handling
+   vmx images built with KIWI and references the following
+   articles:
+
+   * :ref:`vmx`
+
+`Vagrant <http://vagrantup.com>`_ is a framework to
+implement consistent processing/testing work environments based on
+Virtualization technologies. To run a system, Vagrant needs so-called
+**boxes**. A box is a TAR archive containing a virtual disk image and
+some metadata.
+
+To build Vagrant boxes, you can use
+`veewee <https://github.com/jedi4ever/veewee>`_ which builds boxes
+based on AutoYaST, or `Packer <http://packer.io>`_, which is provided
+by Vagrant itself.
+
+Both tools are based on the official distribution media (DVDs). If such
+media does not exist (for example if the distribution is still under
+development or you want to use a collection of your own repositories), the
+KIWI way of building images might be helpful. In addition you can use the
+KIWI image description as source for the
+`Open Build Service <http://openbuildservice.org>`_ which then allows
+building and maintaining boxes in the Build Service as a plus.
+
+A Vagrant box which is able to work with Vagrant has to comply the
+constraints documented in
+`Vagrant Box Constraints <http://docs.vagrantup.com/v2/boxes/base.html>`_.
+Applied to the referenced KIWI image description from :ref:`vmx`
+The following changes are required:
+
+1. Add mandatory packages
+
+   .. code:: xml
+
+      <package name="sudo"/>
+      <package name="openssh"/>
+      <package name="rsync"/>
+
+2. Update the image type setup
+
+   .. code:: xml
+
+      <type image="vmx" filesystem="ext4" format="vagrant" boottimeout="0">
+          <vagrantconfig provider="libvirt" virtualsize="42"/>
+          <size unit="G">42</size>
+      </type>
+
+   This modifies the type to build a vagrant box for the libvirt
+   provider including a pre-defined disk size. The disk size is
+   optional but recommended to provide some free space on disk
+
+3. Add vagrant user
```

Comment:
Are you referring to a user for Vagrant (capital V) or do you mean the specific user `vagrant` (lowercase v)? If you mean the latter, use a specific markup.